evap/templates/base.html

```diff
@@ -160,6 +160,36 @@
         {% endblock %}
     </div>
 </div>
+
+<button id="feedback_button" type="button" class="btn btn-warning" data-toggle="modal" data-target="#feedback_modal">?</button>
+
+<div id="feedback_modal" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="feedback_div">
+    <div class="modal-dialog modal-lg">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"></button>
+                <h4 class="modal-title" id="myModalLabel">Feedback</h4>
+            </div>
+            <div class="modal-body">
+                Do you have any Questions? Or do you want to give us feedback on something that doesn't work or how we could improve this Website? Do it here!
+                <form id="feedback_form">
+                    <div class="form-group">
+                        <label for="sender-email" class="control-label">Email:</label>
+                        <input type="text" class="form-control" id="sender_email">
+                    </div>
+                    <div class="form-group">
+                        <label for="message-text" class="control-label">Question/Feedback:</label>
+                        <textarea class="form-control" id="message_text" style="resize:vertical;"></textarea>
+                    </div>
+                    <button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
+                    <input type="submit" class="btn btn-primary">
+                    <img id="feedback_spinner" src="{% get_static_prefix %}images/spinner.gif" width="30px" height="30px" style="display: none;">
+                </form>
+            </div>
+        </div>
+    </div>
+</div>
```

Comment:
the button probably shouldn't be shown if the user is not authenticated, otherwise someone will somewhen write a script and spam the crap out of us.

also, we could hide it for staff users, they won't use it.